presto-main/src/test/java/com/facebook/presto/operator/scalar/FunctionAssertions.java

```diff
@@ -442,7 +428,7 @@ public static Expression createExpression(String expression, Metadata metadata,
                 SQL_PARSER,
                 symbolTypes,
                 ImmutableList.of(parsedExpression),
-                emptyList(),
+                ImmutableList.of(),
```

Comment:
Just curious why?
It seemed to me that more recent code tends to use `emptyList()` instead of `ImmutableList.of()`